### PR TITLE
Update the Ollama for hosting in notebook

### PR DESCRIPTION
**Pull request by hickeyma**
This updates the details to include how to run Ollama server from within notebook.

### CODE REVIEW

**Comment by fayvor** [comment]
Hi @hickeyma, I'm currently converting this recipe to use Replicate. The idea is to have all recipes written using Replicate by default, with a one-cell copy-and-paste from [LLMs with Langchain](https://github.com/ibm-granite-community/utils/blob/main/recipes/Components/Langchain_LLMs.ipynb) to switch to Ollama (or WatsonX) for the LLM.

**Comment by bjhargrave** [comment]
> I'm currently converting this recipe to use Replicate

Isn't replicate a paid service for which recipe users will need to setup an account and pay? I would think a using ollama for free on their workstation or even in colab would be more inclusive.

File: recipes/Text_to_Shell/Text_to_Shell.ipynb [hunk 1]
```diff
@@ -27,14 +31,22 @@
...
+    "\n",
     "> **NOTE:** By default, this notebook only uses the `3b` model to ensure the widest set of users can run the code. This is also necessary for our CI/CD unit test environment. However, if you have access to a reasonably new and powerful machine, we recommend using the `8b` or the `20b` version instead.\n",
     ">\n",
     "> Change the value in the next cell if you decide to use the `8b` or `20b` model."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (bjhargrave) [inline recipes/Text_to_Shell/Text_to_Shell.ipynb:49]: I don't think you meant to commit this and all following changes.

**Comment by rawkintrevo** [comment]
Yea, we need to add the adr about why we're supporting replicate, not ollama. And why ollama isn't supposed to be in the notebooks.

**Comment by rawkintrevo** [comment]
The replicate_api_token is not included in the ollama notebooks, that's why this is failing.

Again, I don't think this is the way we want to go, but if we do well need a new test workflow that supports ollama AND replicate, but getting away from that is why we broke them up...

File: recipes/Text_to_Shell/Text_to_Shell.ipynb (hunk 1 above)
Comment (hickeyma) [reply recipes/Text_to_Shell/Text_to_Shell.ipynb]: Good catch, not sure what happened there. Fixed now.

**Comment by rawkintrevo** [comment]
Thank you for the contribution @hickeyma . We've decided a while ago that setting up ollama in the notebook created too much clutter, I can refer you to some notebooks where we removed it if you would like to see the implementation. Ollama has a major draw back of only supporting smaller models on most standard machines, while Replicate supports larger models via API call. We understand the hesitancy of paying for usage of large GPUs, and have secured a deal with Replicate to give users free credit. @adampingel can give more details. 

If you would like create a notebook that starts ollama as an example, you'll need to create a new workflow (see [example](https://github.com/ibm-granite-community/granite-code-cookbook/blob/main/.github/workflows/vanilla_workflow.yaml), be sure to change the paths to only reference your new example, and to create a [textfile](https://github.com/ibm-granite-community/granite-code-cookbook/blob/main/.github/notebook_lists/vanilla_notebooks.txt) with the same path) (I know the two text file thing is hacky, I'll be working on that today, it was an expedient solution).

**Review by rawkintrevo — CHANGES_REQUESTED** [review]
Revert this file, create a new example and associate ci/cd workflow.

**Comment by fayvor** [comment]
@hickeyma [here is the PR](https://github.com/ibm-granite-community/granite-code-cookbook/pull/128) where I've replaced Ollama with Replicate in the Text_to_Shell recipe. 

I think this was the first recipe written, so it also needed some updating to more fully embrace Langchain.

**Pull request closed without merging**